src/Component/Provider/ArrayGridProvider.php

```diff
@@ -57,6 +57,6 @@ private function extend(array $gridConfiguration, array $parentGridConfiguration
 
         unset($configuration['extends']);
 
-        return $configuration;
+        return $configuration !== null ? $configuration : [];
```

Comment:
Won't it be the same?
```suggestion
        return $configuration ?? [];
```